wpeverest/everest-forms
# Enhance - Amp plugin compatibility

### All Submissions:






### Changes proposed in this Pull Request:


This PR enhance EVF compatibility with AMP plugin.

### How to test the changes in this Pull Request:

1. Install the Amp plugin (https://amp-wp.org/)
2. Setup  Amp plugin according your requirements.
3. Enable amp in pages and verify EVF Form is working or not accordingly.
4. Test these entire addons also :- Everest Forms (Pro) , Everest Forms - Save and Continue , Survey, Polls and Quiz ,Repeater Fields
5. At last captcha is working or not  in EVF form(only v3 captcha is supported)

### Dependent Branch
https://github.com/wpeverest/everest-forms-pro/pull/451

### Types of changes:




### Other information:




### Changelog entry

>Enhance - Amp plugin compatibility
